speakers/harshvardhan-karn.md

```diff
@@ -0,0 +1,17 @@
+# Harshvardhan Karn
+
+- Company : MayaData
+- GitHub: harshvkarn
+- Title: Software Engineer
+- Location: Global
```

Comment:
Can you add another field called "Residence: Bangalore, India, Asia." In the format of Resident City, Country, Continent.

Reply:
Sure.

Reply:
Done!